server/src/controllers/scraping/index.js

```diff
@@ -0,0 +1,22 @@
+const axios = require('axios');
+const cheerio = require('cheerio');
+const fs = require('fs');
+
+axios.get('https://www.futurelearn.com/courses').then((res) => {
+  const $ = cheerio.load(res.data);
+  const courses = [];
+
+  $('.m-card').each((i, el) => {
+    const title = $(el).find('h4').text();
+    const img = $(el).find('.image-cover_3Epqi').attr('src');
+    const rate = $(el).find('.ReviewStars-text_mSEFD').text();
+    const description = $(el).find('p').text();
+    const link = $(el).find('a').attr('href');
```

Comment:
edit the course url to be valid : 
```suggestion
        const link = `futurelearn.com${$(el).find('a').attr('href')}`;
```